modules/serving-transport-encryption-configuring-selfsigned-clusterissuer.adoc

```diff
@@ -0,0 +1,70 @@
+:_content-type: PROCEDURE
+[id="serving-transport-encryption-configuring-selfsigned-clusterissuer_{context}"]
+= Configuring a SelfSigned ClusterIssuer
+
+[IMPORTANT]
+====
+For simplicity, this procedure uses a `SelfSigned` issuer as root certificate. However, be aware of the implications and limitations of this method documented in the "SelfSigned" section of the `cert-manager` documentation.
+
+If you run your company-specific Private Key Infrastructure (PKI), Red Hat recommends the CA issuer. Refer to the cert-manager documentation on CA issuers for more details. However, you can use any other issuer that allows signing of certificates as described in the "Choice of a certificate issuer" section.
+====
```

Comment:
```suggestion
The following procedure uses a `SelfSigned` issuer as the root certificate. See the link:https://cert-manager.io/docs/configuration/selfsigned/[SelfSigned cert-manager documentation] for information about the implications and limitations of this method.

If you manage your own company-specific Private Key Infrastructure (PKI), use the CA issuer. Refer to the cert-manager documentation on CA issuers for more details.
====
```
Do not use "RH recommends" per https://redhat-documentation.github.io/supplementary-style-guide/#recommend - instead, direct users to take the recommended action. Also, add links to the documentation here when we mention cert-manager docs - there's no reason not to do this as long as the link isn't an xref inside a module. For links to docs.redhat.com or the cert-manager.io docs, links are permitted.